docs/core/linux-prerequisites.md

```diff
@@ -127,244 +155,417 @@ scl enable rh-dotnet20 bash
 **.NET Core 1.1**
 
 Install .NET Core 1.1 SDK and Runtime:
+
 ```bash
 yum install rh-dotnetcore11
 ```
+
 Enable .NET Core 1.1 SDK and Runtime for your environment:
+
 ```bash
 scl enable rh-dotnetcore11 bash
 ```
 
 **.NET Core 1.0**
 
 Install .NET Core 1.0 SDK and Runtime:
+
 ```bash
 yum install rh-dotnetcore10
 ```
+
 Enable .NET Core 1.0 SDK and Runtime for your environment:
+
 ```bash
 scl enable rh-dotnetcore10 bash
 ```
+
 ---
-4. Run the `dotnet --help` command to prove the installation succeeded.
+4. Run the `dotnet --version` command to prove the installation succeeded.
 
      ```bash
-     dotnet --help
+     dotnet --version
```

Comment:
since you already give the command inline they need to run, do you need to have this again?

Reply:
If they want to cut and paste?

Reply:
Fair enough 👍

Reply:
@mairaw - Thank you for another pass of thoughtful comments!